docs/explanations/curator_data_model.md

```diff
@@ -48,9 +48,20 @@ The end goal is to create a JSON Schema that can be used in Curator. A JSON Sche
 
 Note: Individual columns are covered later on this page.
 
+These columns must be present in your CSV data model:
+
+- `Attribute`
+- `DependsOn`
+- `Description`
+- `Valid Values`
+- `Required`
+- `Parent`
+- `Validation Rules`
+
 Defining data types:
 
 - Put a unique data type name in the `Attribute` column.
+- Put the value `DataType` in the `Parent` column.
```

Comment:
Could you confirm whether this step is required? If I remember correctly, the `Parent` column is only used for data visualization, so it shouldn’t be required.

Reply:
oh never mind. I tried without `Parent` column and got an error: 
```
 File "/Users/lpeng/code/synapsePythonClient/synapseclient/extensions/curator/schema_generation.py", line 606, in check_schema_definition
    raise ValueError(
    ...<2 lines>...
    )
ValueError: Schema extension headers: {'DependsOn', 'Format', 'Description', 'Source', 'DependsOn Component', 'Pattern', 'Valid Values', 'Maximum', 'Required', 'Minimum', 'Attribute', 'Properties', 'columnType', 'Validation Rules'} do not match required schema headers: ['Attribute', 'Description', 'Valid Values', 'DependsOn', 'Required', 'Parent', 'Validation Rules']
(
```